src/Filter/ChoiceFilter.php

```diff
@@ -57,7 +56,7 @@ protected function filter(ProxyQueryInterface $query, string $field, FilterData
                 return;
             }
 
-            if ($data->isType(ContainsOperatorType::TYPE_NOT_CONTAINS)) {
+            if ($data->isType(EqualOperatorType::TYPE_NOT_EQUAL)) {
```

Comment:
Both constant are equal to 2.

But since we're using the EqualOperatorType for this filter, it makes more sens to use the constants too